subscription_integration_test.go

```diff
@@ -197,6 +197,10 @@ func TestSubscriptionAllFieldsWithBillingDayOfMonth(t *testing.T) {
 	if x := sub1.Descriptor.URL; x != "example.com" {
 		t.Fatalf("got descriptor url %#v, want example.com", x)
 	}
+	year, month, day := time.Now().Date()
+	if x := sub1.CreatedAt; x.Year() != day || x.Month() != month || x.Day != day {
+		t.Fatalf("got %v %v %v , want %v %v %v", x.Year(), x.Month(), x.Day(), year, month, day)
+	}
```

Comment:
This test could be flaky. It's best if it can just check it's not nil.

Reply:
@leighmcculloch i see. Let me fix that real quick